components/netlify/sources/new-deploy-failure/new-deploy-failure.mjs

```diff
@@ -0,0 +1,23 @@
+import common from "../common/common.mjs";
+import { deployHooks } from "../common/constants.mjs";
+
+export default {
+  ...common,
+  key: "netlify-new-deploy-failure",
+  name: "New Deploy Failure (Instant)",
+  description: "Emit new event when a new deployment fails",
+  type: "source",
+  version: "0.0.2",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getHookEvent() {
+      return deployHooks.DEPLOY_FAILED;
+    },
+    getMetaSummary(data) {
+      const { commit_ref: commitRef } = data;
+      return `Deploy failed for commit ${commitRef}`;
+    },
+  },
+  run: common.methods.run,
```

Comment:
If the `run` method is moved to the component, as mentioned in the comment above, this line can be removed.

```suggestion
```

Reply:
Addressed